README.md

```diff
@@ -0,0 +1,12 @@
+[![Build
+Status](https://travis-ci.org/noironetworks/aci-containers.svg?branch=master)](https://travis-ci.org/noironetworks/aci-containers) [![image](https://coveralls.io/repos/github/noironetworks/aci-containers/badge.svg?branch=master)](https://coveralls.io/github/noironetworks/aci-containers?branch=master)
+
```

Comment:
Not sure it matters, but ideally this should be like this I think:
```
[![Build Status](https://travis-ci.org/noironetworks/aci-containers.svg?branch=master)](https://travis-ci.org/noironetworks/aci-containers)
[![Coverage Status](https://coveralls.io/repos/github/noironetworks/aci-containers/badge.svg?branch=master)](https://coveralls.io/github/noironetworks/aci-containers?branch=master)

```

Reply:
Done!